app/containers/Forms/FuelFields.tsx

```diff
@@ -61,8 +68,18 @@ const FuelFields: React.FunctionComponent<Props> = ({
         </Form.Group>
       </Col>
       <Col xs={12} md={4}>
-        <Form.Group controlId="id.quantity">
-          <Form.Label>Quantity</Form.Label>
+        <FieldTemplate
+          required
+          hidden={false}
+          id="fuel.quantity"
+          classNames="form-group field field-number"
+          label={quantitySchema.title}
+          schema={quantitySchema}
+          uiSchema={uiSchema.quantity || {}}
+          formContext={formContext}
+          help={uiSchema.quantity?.['ui:help']}
+          errors={<ErrorList errors={errorSchema?.Quantity?.__errors as any} />}
```

Comment:
```suggestion
          errors={<ErrorList errors={errorSchema?.quantity?.__errors as any} />}
```

Reply:
Either the change above isn't needed (in which case I'm confused as to why) or there's some missing tests